<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AirlinesPage</class>
 <widget class="QWidget" name="AirlinesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#AirlinesPage{
background-color: rgb(0, 0, 0);
}

QPushButton {
	background: rgb(10,10,10);
	color: rgb(255, 246, 0);
	border: 3px solid #fff600;
	padding: 4px;
}

QPushButton:hover {
	background-color: rgb(255, 246, 0);
	color:  rgb(0, 0, 0);
}

QPushButton:disabled {
	background: rgb(134, 141, 157);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="airlineNameLe">
       <property name="styleSheet">
        <string notr="true">color: gray;
background-color: rgb(0, 0, 0);
font-size:20px;
border :none;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">color: gray;
font-size:18px;</string>
         </property>
         <property name="text">
          <string>Code:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="CodeLe">
         <property name="styleSheet">
          <string notr="true">color: white;
background-color: rgb(0, 0, 0);
font-size:18px;
border: 1px solid yellow;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">color:yellow;</string>
         </property>
         <property name="text">
          <string>List of Pilots:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="pilotPte">
         <property name="styleSheet">
          <string notr="true">color:yellow;
background-color: rgb(39, 39, 39);
border: 1px solid yellow;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 0);</string>
         </property>
         <property name="text">
          <string>List of Flights:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="FlightPte">
         <property name="styleSheet">
          <string notr="true">color:yellow;
background-color: rgb(39, 39, 39);
border: 1px solid yellow;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 0);</string>
         </property>
         <property name="text">
          <string>List of Hosts:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="HostPte">
         <property name="styleSheet">
          <string notr="true">color:yellow;
background-color: rgb(39, 39, 39);
border: 1px solid yellow;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 0);</string>
         </property>
         <property name="text">
          <string>List of Airplanes:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="AirplanePte">
         <property name="styleSheet">
          <string notr="true">color:yellow;
background-color: rgb(39, 39, 39);
border: 1px solid yellow;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="prvBtn">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="subBtn">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Submit Changes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextBtn">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
